i18n/messages.json

```
@@ -351,6 +351,7 @@
   "diff.controls.unifiedView": "Unified View",
   "diffPage.noDocuments": "No documents were selected for comparison.",
   "diffPage.versus": "Vs.",
+  "distributionAnalytics.disabled.message": "Distribution analytics is disabled because current query would impact quality of the service. Please select another path.",
```

```suggestion
  "distributionAnalytics.disabled.message": "Distribution analytics is disabled because the current query would impact the quality of the service. Please select another path.",
```
(according to Grammarly)